src/renderer/themes.css

```diff
@@ -598,14 +598,18 @@ it can be safely elided. This looks quite pleasant on this theme. */
   --primary-color-active: #8d98e4;
 }
 
-.mainSolarizedYellow,
```

Comment:
Yellow's logo is gone
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/4d5301ed-2adc-44c1-9d54-4629d9ca8c93)

Reply:
Thanks for pointing that out! Fixing it!